lib/ios-crash-log.js

```diff
@@ -0,0 +1,61 @@
+import path from 'path';
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+import { asyncmap } from 'asyncbox';
+
+const glob = B.promisify(_glob);
+
+
+require('date-utils');
+const logger = getLogger('IOS-crash-log');
+const CRASH_DIR = path.resolve(process.env.HOME, 'Library', 'Logs', 'DiagnosticReports');
+
+class IOSCrashLog { 
+  constructor() {
+    this.prevLogs = [];
+    this.logsSinceLastRequest = [];
+  }
+
+  async getCrashes () {
+    try {
+      return await glob(`${CRASH_DIR} /*.crash`);
+    } catch (err) {
+      logger.errorAndThrow(`There was a problem getting the crash list: ${err}`);
+      throw err;
```

Comment:
`errorAndThrow` will throw, so this line will never be reached and can be got rid of